agasc/supplement/magnitudes/star_obs_catalogs.py

```diff
@@ -43,6 +44,9 @@ def get_star_observations(start=None, stop=None, obsid=None):
 
     star_obs.add_index(['mp_starcat_time'])
 
+    max_time = CxoTime(events.dwells.all().table[-1]['stop'])
```

Comment:
I don't play much with the django but is something like:
`kadi.events.dwells.all().latest('tstart').stop`
faster?  Not sure if there is a kadi method for this but for the one-off it seems fine.